docs/source/visualisation/visualise_charts_with_plotly.md

```diff
@@ -65,25 +51,37 @@ shuttle_passenger_capacity_plot:
       xaxis_title: Shuttles
       yaxis_title: Average passenger capacity
       title: Shuttle Passenger capacity
+
+shuttle_passenger_capacity_plot:
+  type: plotly.JSONDataSet
+  filepath: data/08_reporting/shuttle_passenger_capacity_plot.json
+
 ```
 
 
-### `plotly.JSONDataSet`
+### Create the template reporting pipeline
 
-To use this dataset, configure your plot in your Kedro node. This dataset supports [Plotly Express](https://plotly.com/python/plotly-express) and [Plotly Graph Objects](https://plotly.com/python/graph-objects/).
+In the terminal, run the following command to generate a template for the reporting pipeline:
 
+```bash
+kedro pipeline create reporting
+```
 
-Below is an example of how to visualise plots using [Plotly Express](https://plotly.com/python/plotly-express) and [Plotly Graph Objects](https://plotly.com/python/graph-objects/) on Kedro-Viz using the `plotly.JSONDataSet`.
+### Add the Plotly reporting nodes
 
-The below functions can be added to the `nodes.py` and `pipeline.py` files respectively.
+Add the following to `src/kedro_tutorial/pipelines/reporting/nodes.py`:
 
 ```python
-# nodes.py
 import plotly.express as px
 import plotly.graph_objs as go
 import pandas as pd
 
-# the below function uses plotly.express
+
+def compare_passenger_capacity(preprocessed_shuttles: pd.DataFrame):
```

Comment:
I still find it a bit confusing that all three functions here are called `compare_passenger_capacity`, it's not entirely clear to me which one is used to create the data defined in the catalog.

Reply:
Thanks @merelcht!

So late last year, if you remember, I found this example had stopped displaying a graphic on Kedro 0.18.4 ([and you tenaciously tracked it down to a bug](https://github.com/kedro-org/kedro-viz/issues/1205).

As part of the analysis (to see if it was underlying Kedro Viz or my own code) I made some changes to the example (but haven't yet dropped them into the PR) to give each node a different name/function name. I found that the Plotly graphic objects works fine on 0.18.3 (that's what I was seeing all along) but Plotly express code doesn't, and probably hasn't for some time. So we now know that the `compare_passenger_capacity` node that was running was the one for graphic objects. 

I'm just about to make a post to Slack to ask for some help in getting the express version working.

Reply:
Here's the post https://kedro-org.slack.com/archives/C03RKP2LW64/p1672757755832089

Until I can fix the code (and get it working on 0.18.4+ as described by https://github.com/kedro-org/kedro-viz/issues/1205) I'll need to put this on hold.

Reply:
That makes sense. This task shows how important keeping your docs up to date is and how it helps finding bugs!